File: Codeforces/1. ICPC Assiut University Community/Contest #1/E. Interval Sweep/main.cpp
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/E
#include<iostream>
using namespace std;
 
int main(){
    short a, b;

    cin >> a >> b;
    cout << (a > 0 && (a == b || a-1 == b || a+1 == b)? "YES" : "NO") << endl;
}

File: Codeforces/1. ICPC Assiut University Community/Contest #1/E. Interval Sweep/main.cs
// https://codeforces.com/group/MWSDmqGsZm/contest/326175/problem/E

using System;

namespace Codeforces.Assuit.Contest1
{
    class ProblemD
    {
        static void Main(string[] args)
        {
            string[] input = Console.ReadLine().Split(' '); // take one line input
            short a = short.Parse(input[0]);
            short b = short.Parse(input[1]); 
            Console.WriteLine((a > 0 && (a == b || a-1 == b || a+1 == b)? "YES" : "NO")); 
        }
    }
}
